### ISSUE TITLE
Script to convert ipynb to HTML

### ISSUE THREAD

**Issue by blackrosedragon2** [Adversarial-Deep-Learning/adversarial-deep-learning.github.io#4]
Given a link to an ipynb file, this script must be able to convert this file into a useable HTML file that will be used to create tutorial pages later on. 

- [ ] Ipynb -> Html Script

**Comment by anekar** [comment]
Start by taking these steps
```
1. select File 
2. Download as 
3.  HTML 
```
OR 
Run the script 
```
jupyter nbconvert  --to html notebook.ipynb
```

**Comment by blackrosedragon2** [comment]
Yes, for single and locally downloaded ipynb notebooks this will work. However, the script should ideally run with:
1. Providing URLs from code-soup's IPYNB files (that will be inside of the tutorials folder)
2. Downloading those ipynb files and converting them as you have mentioned 
3. Store them chapter-wise into the "adl" folder of this repository. 
All of these tasks should occur automatically using a single script. It can be a bash script or python script, anything you're comfortable with.
